### ISSUE TITLE
XvsY error

### ISSUE THREAD

**Issue by chaosphere2112** [UV-CDAT/uvcdat#1620]
```
import cdms2, vcs
x = vcs.init()
f = cdms2.open(vcs.sample_data + "/clt.nc")
x.xvsy(f("v"), f("clt"))
```

gives:
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-2-b691057b570d> in <module>()
----> 1 x.xvsy(v, s)

/Users/fries2/build-uvcdat/install/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/Canvas.pyc in xvsy(self, *args, **parms)
   1705 """
   1706         arglist = _determine_arg_list('xvsy', args)
-> 1707         return self.__plot(arglist, parms)
   1708     xvsy.__doc__ = xvsy.__doc__ % (plot_keywords_doc,
   1709                                    graphics_method_core,

/Users/fries2/build-uvcdat/install/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/Canvas.pyc in __plot(self, arglist, keyargs)
   3498                 elif tp == "default":
   3499                     tp = "boxfill"
-> 3500                 gm = vcs.elements[tp][arglist[4]]
   3501                 if hasattr(gm, "priority") and gm.priority == 0:
   3502                     return

KeyError: '__xvsy_653912073685959_xvsy_'
```

**Comment by doutriaux1** [comment]
Actually this shouldn't work because the shape won't match, but it's definitely crashing first for another reason.

**Comment by williams13** [comment]
No it shouldn’t work, but it shouldn’t crash either. Just doing typical user things…

**Comment by doutriaux1** [comment]
@chaosphere2112 your example at the top still fails for me in master.

**Comment by doutriaux1** [comment]
```
  File "/git/uvcdat/testing/vcs/test_vcs_1d_fails_2_slabs.py", line 5, in <module>
    x.xvsy([1,2,3,4],[1,2,3,4],bg=False)
  File "/lgm/uvcdat/2015-11-09/lib/python2.7/site-packages/vcs/Canvas.py", line 1707, in xvsy
    return self.__plot(arglist, parms)
  File "/lgm/uvcdat/2015-11-09/lib/python2.7/site-packages/vcs/Canvas.py", line 3500, in __plot
    gm = vcs.elements[tp][arglist[4]]
KeyError: '__1d_64410090437423'
```

**Comment by doutriaux1** [comment]
@chaosphere2112 
```
x.xvsy([1,2,3,4],[1,2,3,4],bg=False)
```
that should work

**Comment by doutriaux1** [comment]
@chaosphere2112 I will let you finish #1621 before completing the part where shapes mismatch

**Comment by chaosphere2112** [comment]
@doutriaux1 Ah, I fixed the problem for `.plot`, not the specific graphics-method plot functions. I'll get another PR up for those.

**Comment by aashish24** [comment]
@doutriaux1 @chaosphere2112 can we close this issue?

**Comment by chaosphere2112** [comment]
@aashish24 Not until #1669 is merged.

**Comment by aashish24** [comment]
okay, thanks @chaosphere2112
